sass/woocommerce/_archives.scss

```diff
@@ -83,7 +83,7 @@
 				text-align: center;
 
 				@media (max-width: 768px) {
-					& {
+					@at-root body.responsive#{&} {
```

Comment:
@adiraomj - Not sure why, but I seem to be getting a SCSS syntax error with this. What does this syntax do? Is it something that might have been introduced more recently? Maybe I just need to update my SASS compiler somehow :)